cmd/kured/main.go

```diff
@@ -265,6 +271,9 @@ func flagCheck(cmd *cobra.Command, args []string) {
 	if !reflect.DeepEqual(preRebootNodeLabelKeys, postRebootNodeLabelKeys) {
 		log.Warnf("pre-reboot-node-labels keys and post-reboot-node-labels keys do not match. This may result in unexpected behaviour.")
 	}
+	if !annotateNodes && minRebootPeriod != 0 {
+		log.Fatal("Cannot use --min-reboot-period without --annotate-nodes")
```

Comment:
Please just warn here, that this feature is used without `annotateNodes=true`, no need for fatal.